src/image-quilt.js

```diff
@@ -243,7 +245,14 @@ var QuiltPatch = React.createClass({
       <p><strong>{art.title_short}</strong></p>
     </span>
 
+    const handleKeyDown = function(event){
+      if(event.keyCode === ENTER_KEY_CODE){
+        onClick()
+      }
+    };
+
     return <Link
+      onKeyDown={handleKeyDown}
```

Comment:
Enter key presses were not registering as clicks. You could tab to the image but you weren't able to access it without using the mouse. This allows the enter key to be pressed.